share/revng/abi/Microsoft_x86_cdecl.yml

```diff
@@ -33,7 +33,9 @@ ScalarTypes:
   - Size: 2
   - Size: 4
   - Size: 8
+    AlignedAt: 4
```

Comment:
Is this by deduction or was it properly documented?

Reply:
The documentation for these is not as well structured as I'd hope. So this came from observation first, but I did manage to find a confirmation of the fact that they are aligned at most at register-sized boundaries after the fact. Do you want me to look for it again?

Reply:
No, just curiosity.